packages/amplify_analytics_pinpoint/android/src/main/kotlin/com/amazonaws/amplify/amplify_analytics_pinpoint/AmplifyAnalyticsBridge.kt

```diff
@@ -34,11 +34,11 @@ class AmplifyAnalyticsBridge {
             try {
                 Amplify.addPlugin(AWSPinpointAnalyticsPlugin(context as Application?))
                 LOG.info("Added AnalyticsPinpoint plugin")
+                flutterResult.success(null)
```

Comment:
Nit: Why is this `flutterResult` here only? Was a little confused at first thinking this was different than the other instances of this change due to the naming (and the `MethodChannel.Result` vs `Result`)

Reply:
`flutterResult.error` is being handled by the `handleAddPluginException` utility.